nuxeo-services/login/nuxeo-platform-login-openid/src/main/java/org/nuxeo/ecm/platform/oauth2/openid/OpenIDConnectProvider.java

```diff
@@ -85,18 +86,22 @@
 
     private String accessTokenKey;
 
+    private String userInfoAuth;
+
     private Class<? extends OpenIDUserInfo> openIdUserInfoClass;
 
     public OpenIDConnectProvider(OAuth2ServiceProvider oauth2Provider, String accessTokenKey, String userInfoURL,
             Class<? extends OpenIDUserInfo> openIdUserInfoClass, String icon, boolean enabled,
-            RedirectUriResolver redirectUriResolver, Class<? extends UserResolver> userResolverClass, String userMapper) {
+            RedirectUriResolver redirectUriResolver, Class<? extends UserResolver> userResolverClass, String userMapper,
+            String userInfoAuth) {
```

Comment:
In a hotfix you can't change an API like that, it would break customer code that tried to access it directly (however unlikely that may be). So you must keep a compatible API. I suggest keep your new constructor signature but also re-add the old one, and make it call the old one with a `USER_INFO_AUTH_URL` value for `userInfoAuth`, to keep compatibility.

Reply:
Done